src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { PostInfo } from '../PostInfo';
+import { PreparedPost } from '../../types/PreparedPost';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  preparedPost: PreparedPost[];
+}
+
+export const PostList: React.FC<Props> = ({ preparedPost }) => (
```

Comment:
The prop name should also be updated in the component definition to 'preparedPosts' 🌟